src/app/core/services/devices.service.ts

```diff
@@ -16,10 +16,8 @@ import { DBService } from './db.service';
 import { DomoticzResponse, Switch, Temp } from '@nd/core/models';
 import { Api } from '@nd/core/enums/api.enum';
 
-const isSwitch = (device: any): device is Switch =>
-  device.SwitchType !== undefined;
-const isTemp = (device: any): device is Temp =>
-  device.Temp !== undefined || device.Humidity !== undefined;
+const isSwitch = (device: any): device is Switch => device.SwitchType !== undefined;
+const isTemp = (device: any): device is Temp => device.Temp !== undefined || device.Humidity !== undefined;
```

Comment:
As we know, the format of the code should be easy-read. In this case, the single line format for the constant definition looks like more readable from my aspect (it may depends on personal coding tradition). However it does not matter in this condition. For "Il est dans l'extension Angular Essentials de John Papa", to be honest, I'm not French and trying to understand the comment by translation tool, however I did not understand this sentence correctly. It will be great if you can make some clarifications in English.

Reply:
What I meant is that I personally use the prettier formatter which is part of the VS Code Angular Essentials extensions. It would be nice to have the same formatter to avoid commits of only format changes.